DQMOffline/Hcal/interface/HcalRecHitsAnalyzer.h

```diff
@@ -70,6 +70,8 @@ class HcalRecHitsAnalyzer : public DQMEDAnalyzer {
   double phi12(double phi1, double en1, double phi2, double en2);
   double dPhiWsign(double phi1,double phi2);  
   
+  std::string TopFolderName_;
```

Comment:
Hi @mtosi - sorry for the slow review

please start your variable names with a lower case letter - eg, topFolderName_